lib/actions/rawRequest.js

```diff
@@ -0,0 +1,84 @@
+const axios = require('axios');
+const fs = require('fs');
+const { messages } = require('elasticio-node');
+const { Logger } = require('@elastic.io/component-commons-library');
+const common = require('../common.js');
+const { getSecret, refreshToken } = require('../util');
+
+if (fs.existsSync('.env')) {
+  // eslint-disable-next-line global-require
+  require('dotenv').config();
+}
+
+const logger = Logger.getLogger();
+
+const SALESFORCE_VERSION = common.globalConsts.SALESFORCE_API_VERSION;
+const { REFRESH_TOKEN_RETRIES } = common.globalConsts;
+
+async function processAction(msg, cfg) {
+  this.logger = logger;
```

Comment:
2 questions about logger:
1. Why you use logger from **component-commons-library** instead of using it from **this.logger**?
2. You overwrite **this.logger**, but it looks like it never used further in code.